CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 2.8)
 
 project(synfig-studio)
 
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
```

Comment:
Please change path to `${CMAKE_BINARY_DIR}/output/lib/`

Almost forgot about it.